dockerfiles/grist/ressources/custom.css

```diff
@@ -57,3 +58,35 @@
   background-color: #16b378;
   border-color: #16b378;
 }
+/* END Custom ACL colors */
+
+/* START Custom document card in home page colors */
+.test-dm-public {
+  --icon-color: #cecece !important;
+}
+
+.test-dm-pinned-initials {
+  border-color: #cecece !important;
+}
+/* END Custom document card in home page colors */
+
+/* START Custom tooltip colors */
+.test-tooltip {
+  background-color: rgba(255, 255, 255, 0.8) !important;
+  color: #666666 !important;
+}
+
+.test-tooltip-close [style*=CrossSmall]{
```

Comment:
I am not very comfortable with selectors which inspects the `style` attribute. I propose to take a look together to understand why you need that and if there are other ways what you fix.